src/Opulence/Framework/Databases/Console/Commands/FixMigrationsCommand.php

```diff
@@ -0,0 +1,176 @@
+use Opulence\QueryBuilders\PostgreSql\QueryBuilder as PostgreSqlQueryBuilder;
+use Opulence\QueryBuilders\QueryBuilder as BaseQueryBuilder;
+use RuntimeException;
+
+/**
+ * Defines the command that fixes migrations
+ */
+class FixMigrationsCommand extends Command
+{
+    /** @var string[] The list of migration classes */
+    private $allMigrationClasses = [];
+
+    /** @var string The name of the table to read and write to */
+    protected $tableName = '';
+
+    /** @var IConnection */
+    protected $connection;
+
+    /** @var BaseQueryBuilder */
+    protected $queryBuilder;
+
+    /**
+     * FixMigrationsCommand constructor.
+     *
+     * @param string           $tableName           The name of the table to read and write to
+     * @param string[]         $allMigrationClasses The list of migration classes
+     * @param IConnection      $connection
+     * @param BaseQueryBuilder $queryBuilder
+     */
+    public function __construct(
+        string $tableName,
+        array $allMigrationClasses,
+        IConnection $connection,
+        BaseQueryBuilder $queryBuilder
+    ) {
+        parent::__construct();
+
+        $this->tableName           = $tableName;
+        $this->allMigrationClasses = $allMigrationClasses;
+        $this->connection          = $connection;
+        $this->queryBuilder        = $queryBuilder;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function define()
+    {
+        $this->setName('migrations:fix')
+            ->setDescription('Fixes the migrations database schema');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function doExecute(IResponse $response)
+    {
+        $response->writeln('Fixing migrations database schema...');
+
+        try {
+            $this->connection->beginTransaction();
+
+            switch (get_class($this->queryBuilder)) {
+                case MySqlQueryBuilder::class:
+                    $this->addPrimaryKeyMySql();
+                    break;
+                case PostgreSqlQueryBuilder::class:
+                    $this->addPrimaryKeyPostgreSql();
+                    break;
+                default:
+                    throw new RuntimeException('Unexpected query builder type ' . get_class($this->queryBuilder));
+            }
+
+            $this->connection->commit();
+        } catch (Exception $e) {
+            if ($this->connection->inTransaction()) {
+                $this->connection->rollBack();
+            }
+            $response->writeln(sprintf('<fatal>%s</fatal>', $e->getMessage()));
+        }
+    }
+
+    /**
+     *
+     */
+    protected function addPrimaryKeyMySql(): void
+    {
+        $sql       = sprintf('ALTER TABLE %s ADD COLUMN id int not null FIRST', $this->tableName);
+        $statement = $this->connection->prepare($sql);
+        if (!$statement->execute()) {
+            throw new RuntimeException(sprintf('statement error: %s', json_encode($statement->errorInfo())));
```

Comment:
```suggestion
            throw new RuntimeException(sprintf('Failed to add the ID column: %s', json_encode($statement->errorInfo())));
```

Also, do we know for sure that `errorInfo()` does not leak anything like DB passwords?